rmgpy/data/kinetics/familyTest.py

```diff
@@ -639,6 +640,58 @@ def testSaveFamily(self):
             shutil.rmtree(os.path.join(settings['test_data.directory'], 'testing_database/kinetics/families/intra_H_copy'))
 
 
+class TestLoadFilterFits(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Load an RMG job to test loading the YAML file with custom filter criteria.
+        """
+
+        cls.listener = ConcentrationPrinter()
+
+        folder = os.path.join(os.path.dirname(__file__), '../../solver/files/listener/')
+        inputFile = os.path.join(folder, 'input.py')
+        chemkinFile = os.path.join(folder, 'chemkin/chem.inp')
+        spc_dict = os.path.join(folder, 'chemkin/species_dictionary.txt')
+
+        # set-up RMG object
+        cls.rmg = RMG()
+
+        # load kinetic database and forbidden structures
+        cls.rmg.database = RMGDatabase()
+        path = os.path.join(settings['database.directory'])
+
+        # forbidden structure loading
+        cls.rmg.database.loadForbiddenStructures(os.path.join(path, 'forbiddenStructures.py'))
+        # kinetics family loading
+        cls.rmg.database.loadKinetics(os.path.join(path, 'kinetics'),
+                                       kineticsFamilies=['H_Abstraction'],
+                                       reactionLibraries=[])
+
+        cls.rmg = loadRMGPyJob(inputFile, chemkinFile, spc_dict, generateImages=False, checkDuplicates=False)
+
+    def testLoad(cls):
+        """
+        Test if loading Arrhenius fits of the highest rates in each training reaction family from a file located at
+        `fits_location` on disk works.
+        """
+
+        loadFilterFits(cls.rmg)
+
+        cls.assertIsNotNone(cls.rmg.reactionSystems[0].unimolecularFilterFit)
+        cls.assertIsNotNone(cls.rmg.reactionSystems[0].bimolecularFilterFit)
+
+
+class ConcentrationPrinter(object):
```

Comment:
Define this class in rmgpy.rmg.listener.py and import it from there. This will allow you to delete this here, and also where they occur in simpleTest.py etc.

Reply:
Thanks! Add a blank line at the end of listener.py as well

Reply:
Done.